test/pageobjects/Confirm.js

```diff
@@ -80,6 +81,10 @@ class Confirm extends BasePage {
     this.uploadedVideo.isDisplayed()
     this.driver.executeScript("arguments[0].play();", this.uploadedVideo)
   }
+
+  async waitForConfirmBtnToBeLocated() {
+    this.driver.wait(until.elementLocated(By.css('.onfido-sdk-ui-Confirm-btn-primary')))
```

Comment:
I think this could be added in `utils/mochaw`. In that file you can modify the function on line 5 to accept another parameter, for example:

```
const $driver = driver => (selector, wait) => {
  if (wait) {
    return driver.wait(until.elementLocated(By.css(selector)))
  }
  return driver.findElement(By.css(selector))
}
```
Then in this file, line 6, you can just do 

```
shouldWait = true
get confirmBtn() { return this.$('.onfido-sdk-ui-Confirm-btn-primary', this.shouldWait)}

Reply:
I tried that approach and I didn't see improvement. I noticed more test failures. This is the implementation on the other branch pointing to this one:

https://github.com/onfido/onfido-sdk-ui/compare/improvement/CX-3979-remove-sleep-to-wait-for-elements...improvement/wait-for-ui-elements-one-wait-method?expand=1

Please take a look and let me know if I missed something.

Reply:
I had a look and I think Ricardo is right, we should use implicit wait if possible.
I think you can do something like this when the driver is initialised.
`driver.manage().timeouts().implicitWait(2000)`

Reply:
We already have this in `main.js` 

```
await driver.manage().setTimeouts({
    implicit: 10000,
    pageLoad: 10000
  })
```